src/dataregistry/query.py

```diff
@@ -219,7 +245,14 @@ def find_datasets(self, property_names=None, filters=[]):
                 print(e.StatementError.orig)
                 return None
 
-        return result
+        if return_type == "dict":
```

Comment:
"dict" is not very informative. I prefer something along the lines of "property_dict" or "by_property".   
It appears, at least in a query without a join, that the values returned by `keys()` are just the column name, not prepended by table name, whereas Query.list_dataset_properties( ) returns a list of <table_name>.<column_name>. We should see what keys() returns when a join is involved.